### ISSUE TITLE
Implement DBCC commands for DirectQuery mode

### ISSUE THREAD

**Issue by SharmaAntriksh** [DaxStudio/DaxStudio#819]
Is it possible to send DBCC commands such as DROPCLEANBUFFERS and FREEPROCCACHE etc when testing performance in DirectQuery mode?

**Comment by dgosbell** [comment]
This is not currently possible. But I have an idea for some future functionality which could possibly be used to enable something like this.  I probably would not do anything that is specific to a particular data source, but I'm thinking of maybe something that would let you execute a custom script before running a query or before each cold cache benchmark run.

**Comment by SharmaAntriksh** [comment]
Yup, that's even better, providing control to user to run their scripts depending on the type of database 👍

**Comment by SharmaAntriksh** [comment]
Is it possible to implement this with the current version? It would be nice to issue DBCC commands right before the query execution.

**Comment by SharmaAntriksh** [comment]
@dgosbell bumping
